app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_cmd = command.split()
+    if len(split_cmd) < 3:
+        print("Invalid value")
+        return
+
+    if split_cmd[1] != split_cmd[2]:
```

Comment:
Perform unpacking to avoid using indexing for better code readability and check that the first command is `cp`